ambry-store/src/main/java/com.github.ambry.store/IndexValue.java

```diff
@@ -180,6 +180,12 @@ private IndexValue(long size, Offset offset, byte flags, long expiresAtMs, long
     version = PersistentIndex.CURRENT_VERSION;
   }
 
+  public boolean validateAuthorization(short accountId, short containerId) {
```

Comment:
doesn't belong here - lets put it in a utils file

Reply:
can be as simple as passing two account ids and containers ids and getting them matched if the first account id and first container id are not unknown. Just have to make sure we pass the account/container ID from the store as the "first" account and container id

Reply:
sounds good.